bin/check_samplesheet.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+
+# This script is based on the example at: https://raw.githubusercontent.com/nf-core/test-datasets/atacseq/design.csv
```

Comment:
Nice addition!

Reply:
The code is ready to be used but not implemented yet, since the downstream parts need to be changed to expect meta and read values. I added to code as commented to the workflow